### PR TITLE
Update readMe

### PR DESCRIPTION
**Pull request by MostafaRastegar**

### CODE REVIEW

**Comment by rezaerami** [comment]
thank you for the docs, but by documentation, I meant docz, so we can have a demo also.

**Comment by MostafaRastegar** [comment]
Add mdx

File: packages/ui/packages/Carousel/index.mdx [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+            prevArrow={
+                <Button
+                    icon={<AngleLeftIcon />}
+                    size="xs"
+                    modifier="link"
+                    color="gray"
+                    shade="dark"
+                />
+            }
+            >
```
Comment (sayjeyhi) [inline packages/ui/packages/Carousel/index.mdx:60]: has mdx pasring error:

```bash
   90 |     ....
    91 |     nextArrow={
  
  
    SyntaxError: unknown: Expected corresponding JSX closing tag for <Carousel> (8  8:86)
      86 | `}</code></pre>
      87 | <p>{`next:
    > 88 | add custom icon DOM or Component for nextArrow,prevArrow on `}<Carousel  >{` attribute`}</p>
         |                                                                                         ^
      89 | <pre><code parentName="pre" {...{"className":"language-es6"}}>{`  <Caro  usel
      90 |     ....
      91 |     nextArrow={/home/runner/work/frontend-toolbox/frontend-toolbox/pack  ages/ui/packages/Carousel/README.md: unknown: Expected corresponding JSX closi  ng tag for <Carousel> (88:86)
      86 | `}</code></pre>
      87 | <p>{`next:
    > 88 | add custom icon DOM or Component for nextArrow,prevArrow on `}<Carousel  >{` attribute`}</p>
         |                                                                                         ^
      89 | <pre><code parentName="pre" {...{"className":"language-es6"}}>{`  <Caro  usel
      90 |     ....
      91 |     nextArrow={
```

detailed message [here](https://github.com/snappmarket/frontend-toolbox/pull/186/checks?check_run_id=605308133)

**Review by sayjeyhi — CHANGES_REQUESTED** [review]
your PR has build error

File: packages/ui/packages/Carousel/index.mdx (hunk 1 above)
Comment (MostafaRastegar) [reply packages/ui/packages/Carousel/index.mdx:60]: fixed

**Comment by rezaerami** [comment]
all the build and docz issues are fixed now, but the only issue i see is the navigation buttons
in the RTL slider, when i click on the first page button, it goes to that page but it doesn't activate the related button, also when i use arrow button to go to the first page of slider, it happens again and the first navigation button wont be activated, but when it's auto sliding to that, there's no problem

**Pull request closed without merging**